Wrapping/Generators/Python/itkExtras.py

```diff
@@ -642,8 +642,12 @@ def imread(filename, pixel_type=None, fallback_only=False):
 
     The reader is instantiated with the image type of the image file if
     `pixel_type` is not provided (default). The dimension of the image is
-    automatically found. If the given filename is a list or a tuple, the
-    reader will use an itk.ImageSeriesReader object to read the files.
+    automatically found. If the filename provided is a directory then the
+    directory is assumed to be for a DICOM image and the filename is
+    replaced with list of DICOM filenames within that directory. If the
+    given filename is a list or a tuple (including the case that the
+    filename was initially a DICOM directory), the reader will use an
```

Comment:
rm *(including the case that the filename was initially a DICOM directory)*

Reply:
Will do.